app/models/manageiq/providers/azure/cloud_manager/scanning/job.rb

```diff
@@ -117,10 +121,11 @@ def snapshot_complete
     end
   end
 
-
   # All other signals
-  alias_method :start_snapshot,     :call_snapshot_create
-  alias_method :snapshot_delete,    :call_snapshot_delete
+  alias_method :start_snapshot,  :call_snapshot_create
+  alias_method :snapshot_delete, :call_snapshot_delete
+  alias_method :abort_job,       :process_abort
```

Comment:
So unintuitive ruby behavior of the day, the `VmScan` class defines `alias_method :abort_job,       :process_abort`, this subclass overrides `process_abort` but `signal(:abort_job)` was calling `VmScan#process_abort` unless I also define the alias in the subclass :shrug: